src/freenet/clients/http/QueueToadlet.java

```diff
@@ -44,10 +45,11 @@
 import freenet.client.async.PersistentJob;
 import freenet.client.async.TooManyFilesInsertException;
 import freenet.client.filter.ContentFilter;
-import freenet.client.filter.KnownUnsafeContentTypeException;
 import freenet.client.filter.FilterMIMEType;
+import freenet.client.filter.KnownUnsafeContentTypeException;
```

Comment:
I'm okay with keeping this currently but I'd rather that future commits - where practical - not mix functional and formatting changes.